pgx/src/hooks.rs

```diff
@@ -12,6 +12,16 @@ use crate::prelude::*;
 use crate::{void_mut_ptr, PgBox, PgList};
 use std::ops::Deref;
 
+#[cfg(any(feature = "pg10", feature = "pg11", feature = "pg12", feature = "pg13"))]
+// JumbleState is not defined prior to postgres v14.
+// This opaque type is here to provide an inner type for
```

Comment:
This can be externally constructed and is thus not really "opaque".
```suggestion
// This zero-sized type is here to provide an inner type for
```